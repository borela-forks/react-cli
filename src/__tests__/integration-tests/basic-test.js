// @flow strict

import React from "react";
import TestRender from "./test-render";

TestRender("Renderer should output text", <div>Basic test</div>);
