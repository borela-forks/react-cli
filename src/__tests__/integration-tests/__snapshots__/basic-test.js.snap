// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renderer should output text 1`] = `
"Basic test++++++++++++++++++++++++++++++++++++++++
"
`;
